### ISSUE TITLE
train loss = nan

### ISSUE THREAD

**Issue by meisu1** [LeiWangR/cl#1]
When I try to train with AMCLSimCLR on CIFAR10 dataset, the loss value is always nan, moreover under src/models.AMCLSimCLRModle, in 'self.criterion = NTXentLoss()' NTXentLoss is not defined, is it also 'lightly.loss.NTXentLoss()'?
